dired.sublime-settings

```diff
@@ -33,6 +33,13 @@
   //     "git_path": ""
   "git_path": "git",
 
+  // Path to hg executable.
+  // Wildcards are supported, e.g. if you use TortoiseHg it would be:
+  //     "hg_path": "%ProgramFiles%/TortoiseHg/hg.exe"
```

Comment:
Well, if it’s too much, just let me know and I’ll do it myself tomorrow. All in all it could be merge right away, but those 3 points above should be done before merge.